src/caselawclient/Client.py

```diff
@@ -646,6 +649,23 @@ def user_can_view_unpublished_judgments(self, username):
         result = multipart_data.parts[0].text
         return result.lower() == "true"
 
+    def user_has_role(self, username, role):
+        vars = {
+            "user": username,
+            "role": role,
+        }
+        return self._send_to_eval(vars, "user_has_role.xqy")
+
+    @cached
+    def user_has_admin_role(self, username):
```

Comment:
I'd be tempted to reword this so that the decoder parts are inside `user_has_role` which can then return a clean True/False answer

Reply:
I'm in two minds about this, as some of the methods in the client return a response and some return strings, and there's no rhyme or reason about which ones do which. Most seem to return a response which is then parsed by something else.